src/main/java/seedu/address/logic/commands/iteration/AddIterationCommand.java

```diff
@@ -66,20 +70,24 @@ public CommandResult execute(Model model, Storage...storage) throws CommandExcep
                     activeCommission.getTitle().toString()));
         }
 
-        String src = toAdd.getImagePath().path;
-        String dst = storage[0].saveIterationImage(src);
+        try {
+            String src = toAdd.getImagePath().path;
+            String dst = storage[0].saveIterationImage(src);
 
-        Iteration toAdd2 = new Iteration(
-                toAdd.getDate(),
-                toAdd.getDescription(),
-                new ImagePath(dst),
-                toAdd.getFeedback()
-        );
+            Iteration toAdd2 = new Iteration(
```

Comment:
Should we change the name of this variable? (Code quality - Use Name to Explain ([Don’t use numbers or case to distinguish names.](https://nus-cs2103-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain:~:text=Don%E2%80%99t%20use%20numbers%20or%20case%20to%20distinguish%20names.)))